### PR TITLE
allow data when performing an action

### PR DESCRIPTION
**Pull request by jamesgorrie**
Simple - effective.
I wasn't 100% sure how to test for it - but happy to work through it with you to find out.

### CODE REVIEW

**Comment by jamesgorrie** [comment]
@theefer Small hack.

File: src/theseus/resource.js [hunk 1]
```diff
@@ -209,15 +209,18 @@ export class Resource {
   /**
    * @return {Promise[Resource|Any]}
    */
-  // TODO: allow passing params and body
-  perform(name) {
```
Comment (theefer) [inline src/theseus/resource.js]: You probably want to default `parameters` to `{}` else `perform('foo')` may throw.

Comment (jamesgorrie) [inline src/theseus/resource.js]: Ah, I thought I had. Done now.

**Comment by theefer** [comment]
:+1:

**Comment by theefer** [comment]
Published as 0.5.2.

**Comment by jamesgorrie** [comment]
Thanks!

**Pull request merged**